lib/iris/fileformats/pp_save_rules.py

```diff
@@ -44,6 +44,17 @@ def _basic_coord_system_rules(cube, pp):
     if cube.coord_system("GeogCS") is not None or cube.coord_system(None) is None:
         pp.bplat = 90
         pp.bplon = 0
+
+        if iris.FUTURE.lam_pole_offset:
+            try:
+                # LAMs should have bplon of 180
+                if (
+                    not cube.coord(axis="x", dim_coords=True).circular
+                    or not cube.coord(axis="y", dim_coords=True).circular
+                ):
+                    pp.bplon = 180.0
+            except iris.exceptions.CoordinateNotFoundError:
+                pass
```

Comment:
Just noticed that we're missing test coverage for this scenario.

Reply:
I've added a "zonal slice" test that should cover it; 6c7cdad06